docs/migration/migrate_6_0_index.asciidoc

```diff
@@ -0,0 +1,337 @@
+[[migrating-6.0-index]]
+== Migrating Kibana index to 6.0
+
+[IMPORTANT]
+==============================================
+The following will make transformations to your Kibana index. It is recommended you backup your Kibana index prior to running any of the following commands.
```

Comment:
Maybe a link to the snapshot/restore docs where it says "backup your Kibana index" ?

https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-snapshots.html